ChangeLog.md

```diff
@@ -6,8 +6,12 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Added
 - Add Annotations API as a service
 
+### Security
+- Bump certifi minimum version to 2023.7.22
```

Comment:
@markus-liang-lrn  Can you ask Edwin to review this Thanks?

Reply:
Hi there @bhavya-shukla-lrn and @markus-liang-lrn , I reviewed. Looks great, guys. Thanks for making this fix to the Python SDK!